transport/wseb/src/test/java/org/kaazing/gateway/transport/wseb/WsebTransportTest.java

```diff
@@ -79,12 +86,7 @@
 	private NioSocketAcceptor tcpAcceptor;
 	private HttpAcceptor httpAcceptor;
 	private WsebAcceptor wsebAcceptor;
-
-    @BeforeClass
-    public static void debugging() throws Exception {
-        PropertyConfigurator.configure("src/test/resources/log4j-trace.properties");
-    }
-
+	
```

Comment:
Is it possible to use the new test rule for WsebConnector here ? Just for my understanding, no need to change the code here.